sllurp/cli.py

```diff
@@ -45,6 +45,9 @@ def cli(debug, logfile):
               help="Tag Population value (default 4)")
 @click.option('-r', '--reconnect', is_flag=True, default=False,
               help='reconnect on connection failure or loss')
+@click.option('-ra', '--reconnect-attempts', type=int, default=5,
+              help='num of reconnect attempts on connection failure or loss. '
+              'Set to -1 for unlimited attempts')
```

Comment:
With a single "-" (minus), you can just have one letter.
As it should not be a common option, it could probably be ok to not have a "short" argument (single minus), but just a long argument.

This is just my opinion, and so up to debate, but maybe "reconnect-retries" might be clearer as it is more common to use the word "retry" for that kind of argument than "attempt".

By the way, in the document you can add the word "Maximum" to say that it is the maximum number of attempts.

Reply:
Ok i'll remove the short args. I was debating between 'reconnect-retries' and 'reconnect-attempts' as well. If you think 'reconnect-retries' is more clear then I'm good with that.